docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -659,25 +689,23 @@ network="rinkeby"
 Predefined network configuration.
 The default is `mainnet`.
 
-Possible values are :
+Possible values are:
 
-`mainnet`
-:   Main Ethereum network
+| Network   | Description                                                                   |
+|-----------|-------------------------------------------------------------------------------|
+| `classic` | Ethereum Classic network                                                      |
+| `dev`     | PoW development network with a very low difficulty to enable local CPU mining |
+| `goerli`  | PoA test network using Clique                                                 |
+| `kotti`   | Ethereum Classic PoA test network using Clique                                |
+| `mainnet` | Main Ethereum network                                                         |
+| `rinkeby` | PoA test network using Clique                                                 |
+| `ropsten` | PoW test network similar to current main Ethereum network                     |
 
-`ropsten`
-:   PoW test network similar to current main Ethereum network. 
-
-`rinkeby`
-:   PoA test network using Clique.
-
-`goerli`
-:   PoA test network using Clique.
-
-`dev`
-:   PoW development network with a very low difficulty to enable local CPU mining.
-
-!!!note
-    Values are case insensitive, so either `mainnet` or `MAINNET` works.
+!!!tip
+    Values are case insensitive, so either `mainnet` or `MAINNET` works. 
+    
+!!!important
+    The [`--network`](#network) and [`--genesis-file`](#genesis-file) option cannot be used at the same time.
```

Comment:
This tip is duplicated - one needs removing and an _option_ changed to _options_ in the tip that's retained

Reply:
Removed extra note and changed _option_ to _options_.